01_install_requirements.sh

```diff
@@ -43,3 +48,13 @@ fi
 cd $SCRIPTDIR
 cp -rv /usr/share/openstack-tripleo-heat-templates ./tripleo-heat-templates
 
+# Patch to make octavia work in standalone:
+# https://review.openstack.org/#/c/628957/6
+pushd tripleo-heat-templates
+cat ../202ca5a_octavia_patch.diff | patch -p1
```

Comment:
What do you think about running `git init` in tht and then apply the patch? @hardys proposed something like this in one of my previous PRs as it's common to have to modify tht and save changes.

Reply:
I think it's a good idea if it works :).  I'd consider just merging this and changing it over after tho.  Unless you want to update it.. I'm done for today :).

Reply:
We can wait until tomorrow :smile: 

What I like of the `git init` workflow is that we won't have to carry `.patch` files around in ocp-doit.